test/tests/meshgenerators/subdomain_element_ids/subdomain_elem_ids_test.i

```diff
@@ -0,0 +1,80 @@
+[Mesh]
+  [gmg]
+    type = GeneratedMeshGenerator
+    dim = 2
+    nx = 10
+    ny = 10
+    xmax = 1
+    ymax = 1
+    subdomain_ids = '
+      0 0 0 0 0 0 0 0 0 0
+      0 0 0 0 0 0 0 0 0 0
+      1 1 1 1 1 1 1 1 1 1
+      1 1 1 1 1 1 1 1 1 1
+      2 2 2 2 2 2 2 2 2 2
+      2 2 2 2 2 2 2 2 2 2
+      3 3 3 3 3 3 3 3 3 3
+      3 3 3 3 3 3 3 3 3 3
+      4 4 4 4 4 4 4 4 4 4
+      4 4 4 4 4 4 4 4 4 4'
```

Comment:
use a CMG at this point
I dont think we want this

Reply:
Do you mean CartesianMeshGenerator?

Reply:
yeah